### DISCUSSION TITLE
Manifest is missing in it

### DISCUSSION THREAD

**Discussion by abhishekgoyalp** [felladrin/linkedin-autoconnect-chrome-extension#616] [General]
Manifest.json file is missing in this project

**Comment by felladrin** [comment]
Hi, @abhishekgoyalp!

Thank you for taking the time to check it out.

The repository is indeed not in the extension format. Please note that the instructions in the Readme indicate that, to use the extension, users should download the .zip file from the Releases page, which has it pre-built.

But you can also build it yourself. Since you have already cloned the repo and have Node.js installed, you can run npm ci && npm run build. This will create the extension correctly structured inside the dist folder.

Kindly let me know if this resolves the issue.
